frontend/packages/git-service/src/utils/import-strategy-detector.ts

```diff
@@ -62,7 +64,17 @@ export const detectImportStrategies = async (
 
   if (repositoryStatus === RepoStatus.Reachable) {
     try {
-      const { files } = await gitService.getRepoFileList();
+      const { files } = await gitService.getRepoFileList({ includeFolder: true });
+
+      if (isRepositoryEnabled) {
```

Comment:
@Lucifergene We should add this only when PAC is enabled and the git repo has a `.tekton` folder with atleast one PLR yaml file.

Reply:
So, in the case of Gitlab, in order to find out `.tekton` folder exists or not, we need this arg. Therefore, without this argument in case of Gitlab, its not possible to understand whether `.tekton` folder exists or not.

Reply:
One thing, that we can do is that, check whether the GitProvider is Gitlab. If so, only then, we can pass the argument. But again, that will be of no use, since the argument is only read by the Gitlab service only.

Reply:
```suggestion
      if (isRepositoryEnabled) {
```
@Lucifergene I am talking about this line. We should add pac strategy only when pac is enabled and .tekton folder with atleast one PLR yaml file is present.